chef-windows.gemspec

```diff
@@ -0,0 +1,22 @@
+gemspec = eval(IO.read(File.expand_path("../chef.gemspec", __FILE__)))
+
+gemspec.platform = Gem::Platform.new ["universal", "mingw32"]
```

Comment:
Mostly my style comment was around putting parentheses in there, more than dropping the nil. I just saw you could in testing/reading the docs.

You're passing an array as a single argument and my brain has to stop to understand that without the parentheses.

```
gemspec.platform = Gem::Platform.new(["universal", "mingw32"])
```